wordpress-mobile/WordPress-iOS
# ULS: update Auth to use functional Password view

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/352
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/363

To test:

---
Prerequisites:
- Enable the `unifiedGoogle` feature flag.
- A Google account that:
  - Is disconnected from WP.com. WP.com > Me > Security > Social Logins > Google > disconnect.
  - Has a WP password set.
  - Has WP 2FA set.

---
- Select `Continue with Google` from either login or signup.
- On the Password view:
  - Enter an invalid password. Verify an error is displayed.
  - Tap the `Rest your password`. Verify the reset password page is displayed.

| Invalid Password | Reset Password |
|--------|-------|
| ![invalid_password](https://user-images.githubusercontent.com/1816888/89467998-d67c5d80-d733-11ea-9168-8773993a5a6c.png) | ![reset_password](https://user-images.githubusercontent.com/1816888/89468018-dda36b80-d733-11ea-93cf-6dbf12001fc8.png) |

---
- Enter a valid password. Verify the Continue button shows loading.
- Verify the new 2FA view is displayed.
- Enter a 2FA code. Verify the Continue button shows loading.

<kbd>![2fa](https://user-images.githubusercontent.com/1816888/89468564-08da8a80-d735-11ea-9c65-9af98b8818d8.png)</kbd>

---
PR submission checklist:
